import-service/handlers/importProductsFile.ts

```diff
@@ -0,0 +1,36 @@
+import { APIGatewayProxyHandler, APIGatewayProxyResult } from 'aws-lambda';
+import 'source-map-support/register';
+import { CORS_HEADERS } from '../common/constants';
+
+const AWS = require('aws-sdk');
+const BUCKET = 'rss-aws-task5';
+
+export const importProductsFile: APIGatewayProxyHandler = async (event) => {
+  console.log('Lambda function importProductsFile has invoked');
+  console.log('event: ', event);
+  console.log('event.queryStringParameters: ', event.queryStringParameters);
+
+  const { name } = event.queryStringParameters;
+  const filePath = `uploaded/${name}`;
+
+  const s3 = new AWS.S3({ region: 'eu-west-1' });
+
+  const params = {
+    Bucket: BUCKET,
+    Key: filePath,
+    Expires: 60,
+    ContentType: 'text/csv'
+  };
+
+  return new Promise<APIGatewayProxyResult>((resolve, reject) => {
+    s3.getSignedUrl('putObject', params, (error, url) => {
+      if (error) return reject(error);
+
+      resolve({
+        statusCode: 200,
```

Comment:
Seems like by REST better to use 202 (Accepted) status code here.

Reply:
As I read from https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html:
> The HTTP status code is in the 200 range for a successful request. For the RequestResponse invocation type, this status code is 200. For the Event invocation type, this status code is 202. For the DryRun invocation type, the status code is 204.